tests/plots/test_summary.py

```diff
@@ -123,16 +123,18 @@ def test_random_summary_with_log_scale():
     return fig
 
 
-def test_summary_plot_with_multiclass_model():
+@pytest.mark.parametrize("background", [True, False])
+def test_summary_binary_multiclass(background):
     # See GH #2893
     lightgbm = pytest.importorskip("lightgbm")
-    num_examples, num_features = 20, 3
-    X = np.random.uniform(size=[num_examples, num_features])
-    y = np.random.choice([0, 1], size=num_examples)
+    num_examples, num_features = 100, 3
```

Comment:
Not strictly related to the scatterplot, but I noticed this test was throwing warnings due to zero SHAP values being plotted, so I fixed it up by increasing the number of examples slightly.